releng/sap-cf/pom.xml

```diff
@@ -149,6 +149,13 @@
         </exclusion>
 			</exclusions>
 		</dependency>
+
+    <!-- Mail -->
+    <dependency>
+      <groupId>com.sap.xsk</groupId>
+      <artifactId>xsk-modules-engines-mail-destination</artifactId>
+      <version>0.14.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
No need to be explicitly added here. It would be better if this dependency is inside the 
**engine-all** module: https://github.com/SAP/xsk/blob/main/modules/engines/engine-all/pom.xml